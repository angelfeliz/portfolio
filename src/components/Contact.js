import React from 'react'

const Contact = () => {
  return (
    <section className="contact">
      <div className="contact-flex-row">
        <div>
          I you like to contact me you can send me a email or linkedin, I will
          be catching up with you soon as posible.
        </div>
        <div className="contact-flex-col">
          <div>
            <i className="fa fa-envelope-o" aria-hidden="true" />
            <span>erodriguezfeliz@gmail.com</span>
          </div>
          <div>
            <i className="fa fa-linkedin-square" aria-hidden="true" />
            <span>https://www.linkedin.com/in/erodriguezfeliz/</span>
          </div>
        </div>
      </div>
    </section>
  )
}

export default Contact
