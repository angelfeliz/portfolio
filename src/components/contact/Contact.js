import React from 'react'
import './contact.css'

const Contact = () => {
  return (
    <section className="contact">
      <div className="contact-flex-row">
        <div className="container_image_footer">
          <p>Think for passing by my portfolio.</p>
        </div>
        <div className="contact-flex-col">
          <div>
            <a
              href="https://stackoverflow.com/users/2777253/angel"
              target="_blank"
              rel="noopener noreferrer"
            >
              <i className="fa fa-stack-overflow fa-3x" aria-hidden="true" />
            </a>
          </div>
          <div>
            <a
              href="https://github.com/angelfeliz"
              target="_blank"
              rel="noopener noreferrer"
            >
              <i className="fa fa-github-square fa-3x" aria-hidden="true" />
            </a>
          </div>
          <div>
            <a
              href="https://www.linkedin.com/in/erodriguezfeliz/"
              target="_blank"
              rel="noopener noreferrer"
            >
              <i className="fa fa-linkedin-square fa-3x" aria-hidden="true" />
            </a>
          </div>
        </div>
        <div className="contact-paragraph">
          <p>Hope to hear from you! Best regards.</p>
        </div>
      </div>
    </section>
  )
}

export default Contact
