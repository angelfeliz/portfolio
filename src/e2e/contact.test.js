import React from 'react'
import Contact from '../components/Contact'
import renderer from 'react-test-renderer'

describe('Test contact component', () => {
  it('renders correctly', () => {
    const rendered = renderer.create(<Contact />)
    expect(rendered.toJSON()).toMatchSnapshot()
  })
})
