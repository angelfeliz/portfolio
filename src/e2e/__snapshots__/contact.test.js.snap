// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test contact component renders correctly 1`] = `
<section
  className="contact"
>
  <div
    className="contact-flex-row"
  >
    <div>
      <p>
        CHeck out!!
      </p>
    </div>
    <div
      className="contact-paragraph"
    >
      <p>
        This is my portfolio so far I hoope you've liked it was created with
                    love and music. You can contact me through my social media networks
                    for any questions suggestions or any topics like sports photography
                    science innovation and projects to improve the quality of life of
                    humanity.
      </p>
      <p>
        Hope to hear from you! Best regards.
      </p>
    </div>
    <div
      className="contact-flex-col"
    >
      <div>
        <i
          aria-hidden="true"
          className="fa fa-envelope-o"
        />
        <span>
          erodriguezfeliz@gmail.com
        </span>
      </div>
      <div>
        <i
          aria-hidden="true"
          className="fa fa-linkedin-square"
        />
        <span>
          https://www.linkedin.com/in/erodriguezfeliz/
        </span>
      </div>
    </div>
  </div>
</section>
`;
